baselines/constraint/constraint_wrapper.py

```diff
@@ -125,16 +127,21 @@ def step(self, action):
 
         Returns
         -------
-        ob: Tensor?
-            An Tensor? or data structure that 
+        ob: Object (Numpy Array)
+            Basically a Numpy array wrapped in 
+            another data structure that 
             is augmented using the augment_obs
             function after being reset.
-        rew: 
+        rew: float
             The reward provided by the environment.
-        done: Boolean?
+        done: Boolean
             Whether or not the episode is finished.
-        info: ??
-            Unclear.
+            Environments reset when episodes finish.
+        info: dict
+            Fetched from OpenAI Gym documentation.
```

Comment:
What does this mean?

Reply:
It was more a note to myself for where the information was originally found.
I can remove it if it's unnecessary.

Reply:
It was a note for where the documentation was originally found (and I guess an indicator that this part isn't changed by your code.)
I can remove it if it's unnecessary.